clients/web/src/stylesheets/layout/global.styl

```diff
@@ -88,6 +88,9 @@ ul.dropdown-menu
 .modal-backdrop.in
   opacity .35
 
+blockquote p
+  font-size inherit
```

Comment:
Bootstrap has a whole bunch of default element styling, among which is setting `blockquote p` to have a font size of 17.5px, which is much larger than the other text rendered in the markdown. This overrides that because it's silly.